DuckDuckGo/ContentBlocker/Mocks/ContentBlockingMock.swift

```diff
@@ -30,8 +30,8 @@ final class ContentBlockingMock: NSObject, ContentBlockingProtocol, AdClickAttri
         var embeddedDataEtag: String = ""
         var embeddedData: Data = .init()
     }
-    var trackerDataManager = TrackerDataManager(etag: ConfigurationStore.shared.loadEtag(for: .trackerDataSet),
-                                                data: ConfigurationStore.shared.loadData(for: .trackerDataSet),
+    var trackerDataManager = TrackerDataManager(etag: ConfigurationStore().loadEtag(for: .trackerDataSet),
+                                                data: ConfigurationStore().loadData(for: .trackerDataSet),
```

Comment:
I'm going to opt to leave these as is. These are instantiating properties on the tests so even adding a dedicated property for the store throws an error in Xcode probably because of init order. Refactoring this to simplify it doesn't seem worth it.